example-get-started/code/src/evaluate.py

```diff
@@ -29,40 +29,7 @@
 predictions_by_class = model.predict_proba(x)
 predictions = predictions_by_class[:, 1]
 
-precision, recall, prc_thresholds = metrics.precision_recall_curve(labels, predictions)
-fpr, tpr, roc_thresholds = metrics.roc_curve(labels, predictions)
-
-avg_prec = metrics.average_precision_score(labels, predictions)
-roc_auc = metrics.roc_auc_score(labels, predictions)
-
-with open(scores_file, "w") as fd:
-    json.dump({"avg_prec": avg_prec, "roc_auc": roc_auc}, fd, indent=4)
-
-# ROC has a drop_intermediate arg that reduces the number of points.
-# https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_curve.html#sklearn.metrics.roc_curve.
-# PRC lacks this arg, so we manually reduce to 1000 points as a rough estimate.
-nth_point = math.ceil(len(prc_thresholds) / 1000)
-prc_points = list(zip(precision, recall, prc_thresholds))[::nth_point]
-with open(prc_file, "w") as fd:
-    json.dump(
-        {
-            "prc": [
-                {"precision": p, "recall": r, "threshold": t}
-                for p, r, t in prc_points
-            ]
-        },
-        fd,
-        indent=4,
-    )
-
-with open(roc_file, "w") as fd:
-    json.dump(
-        {
-            "roc": [
-                {"fpr": fp, "tpr": tp, "threshold": t}
-                for fp, tp, t in zip(fpr, tpr, roc_thresholds)
-            ]
-        },
-        fd,
-        indent=4,
-    )
+live.log_plot("precision_recall", labels, predictions)
```

Comment:
probably it doesn't factor in this https://github.com/iterative/example-repos-dev/pull/102/files#diff-c9e81175bafc05fa2504a715d85f4fb99493314d93b2ba5b81f79fcc77cd09a7L41-L45 , right? that would make Studio slow ... and unfortunately `sklearn` doesn't support dropping intermediate values yet https://github.com/scikit-learn/scikit-learn/issues/21825